server/core/config/raw/step.go

```diff
@@ -37,16 +37,16 @@ const (
 //
 // 2. A map for an env step with name and command or value, or a run step with a command and output config
 //   - env:
```

Comment:
These white space proposed changes seem mistaken

Reply:
I was surprised too, but the linter complained about these:

```sh
server/core/config/raw/step.go:40: File is not `gofmt`-ed with `-s` (gofmt)
//       name: test
//       command: echo 312
//       value: value
```